jsapp/js/components/submissions/table.es6

```diff
@@ -450,13 +501,30 @@ export class DataTable extends React.Component {
           });
       }
 
+      let columnClassName = '';
+      if (
+        (q && NUMERICAL_TYPES.includes(q.type)) ||
+        NUMERICAL_COLUMNS.includes(key)
+      ) {
+        columnClassName += 'rt-numerical-value';
+      }
+
+
+      let columnIcon = null;
+      if (q && q.type) {
+        columnIcon = renderQuestionTypeIcon(q.type);
+      }
+
       columns.push({
         Header: () => {
           const columnName = _this.getColumnLabel(key, q, qParentG);
           const columnHXLTags = _this.getColumnHXLTags(key);
           return (
             <React.Fragment>
-              <span className='column-header-title' title={columnName}>{columnName}</span>
+              <span className='column-header-title' title={columnName}>
```

Comment:
Because of how `react-table` is built the tip is being cut by `overflow`s and by rows below. I made a separate issue for this https://github.com/kobotoolbox/kpi/issues/3286